content/docs/contributing/to-docs/get-started.md

```diff
@@ -1,108 +1,231 @@
-
-   2. When addressing comments, maintain the relative conversation in the PR, edit the topics as necessary, and commit the changes to your fork/branch. Your new commits will be automatically added to the PR.
-
-   3. Make sure to request a re-review of your new commits if required. 
-
-1. **The PR is merged and closed:** When the comments and suggestions have been addressed, and two reviewers have approved the PR, it can be merged into `o3de.org/main`. **Never** merge your own PRs, the final reviewer to sign off is responsible for the merge. The PR is automatically closed when merged.
-
-1. **The related issue is closed:** If you have included a related issue number in the title of your PR, the related issue will be automatically closed when the PR is closed. Your contribution is complete. Congrats!
+---
+linktitle: Get Started
+title: Get Started Contributing to Docs
+description: Get started contributing to Open 3D Engine (O3DE) docs by setting up your local repo, setting up your writing environment, and submitting docs. 
+toc: true
+weight: 200
+---
+
+This section will help you get started on contributing to Open 3D Engine (O3DE) documentation. You'll learn how to set up your writing environment and about the processes you need to get your documentation from text editor to the official O3DE documentation. 
+
+O3DE documentation uses the [fork and pull model](https://en.wikipedia.org/wiki/Fork_and_pull_model) for contributions. As a contributor, you maintain a fork (your own repository of the O3DE documentation) on GitHub, and work locally to edit documentation. You then submit PRs from your fork for review.
+
+{{< note >}}
+The steps on this page use the terminal to run Git commands. You may complete these steps through an alternative tool of your choice, but ensure that you're completing the correct operations. 
+{{< /note >}}
+
+## Prerequisites
+
+* Sign up for a GitHub account here [Join GitHub](https://github.com/join?ref_cta=Sign+up).
+
+* Install **Git** version control software. Get Git here [Git Downloads](https://git-scm.com/downloads).
+
+* Install an editor for making changes to Markdown (`.md`) files. You can use any editor, but we do recommend one that supports Markdown linting. VS Code is commonly used by contributors and you can get it here [Microsoft VS Code](https://code.visualstudio.com/download).
+
+
+## Setting up a local o3de.org repo
+
+In this section, you'll learn to create your own fork of the O3DE documentation. 
+
+
+### Create a fork
+
+A *fork* is your own copy of `o3de.org:main` on GitHub. You can do anything you like within your fork, though it is recommended that you keep your fork synced with `o3de.org:main`, and work within branches in your fork. Working this way ensures the integrity of `o3de.org:main` and makes it easy for you to work at your own pace, experiment with changes, and collaborate with other contributors. 
```

Comment:
Do we need to cover the difference between o3de.org:main and o3de.org:development here?

Reply:
Yep! Creating a section to distinguish the upstream branches. Also amending this section, the fork operation occurs on the whole repo, and i don't believe is dependent on the branch.